src/python/platforms/android/adb.py

```diff
@@ -371,16 +372,33 @@ def recreate_gce_device():
   stop_gce_device()
 
   # Delete all existing images.
-  image_dir = cvd_dir
-  for image_file_path in glob.glob(os.path.join(image_dir, '*.img')):
-    shell.remove_file(image_file_path)
-
-  # Restore images from backup.
-  backup_image_dir = os.path.join(cvd_dir, 'backup')
-  for image_filename in os.listdir(backup_image_dir):
-    image_src = os.path.join(backup_image_dir, image_filename)
-    image_dest = os.path.join(image_dir, image_filename)
-    shell.copy_file(image_src, image_dest)
+  cvd_address = get_gce_address()
+  rm_cmd = 'rm -rf {cvd_dir}/*'.format(cvd_dir=cvd_dir)
```

Comment:
just use f-strings. i.e. f'rm -rf {cvd_dir}/*' 

same for other cases in this file.

Reply:
Done